tests/Disable.test.ts

```diff
@@ -1,5 +1,7 @@
+import {describe, beforeEach, it, expect}     from '@jest/globals';
 import {Filename, ppath, xfs, npath}          from '@yarnpkg/fslib';
-import {delimiter}                            from 'path';
+import {delimiter}                            from 'node:path';
+import process                                from 'node:process';
```

Comment:
These changes shouldn't be necessary, these are all global by default.

Reply:
It's indeed not necessary, more like a personal preference.

Reply:
Fair enough, I thought at first it was necessary for this PR of the output of https://github.com/nodejs/corepack/actions/runs/5022431482/jobs/9005884303, turns out it was not and I left them because I liked them. My bad.